tests/wonderland/js/package.json

```diff
@@ -10,6 +10,6 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@zestymarket/wonderland-sdk": "^1.0.0"
+    "@zestymarket/wonderland-sdk": "^1.4.4"
```

Comment:
Is this what's necessary for Sorskoot to use beacons?

Reply:
I just caught it while I was working, it's in one of our test projects, don't know how I didn't notice it until now. Sorskoot should be able to use beacons with the version on NPM right now

Reply:
No worries, appreciate the fix!